src/OpenLoco/Windows/Options.cpp

```diff
@@ -1844,7 +1844,7 @@ namespace OpenLoco::Ui::Options
 
     namespace Misc
     {
-        static const Gfx::ui_size_t _window_size = { 420, 139 };
+        static const Gfx::ui_size_t _window_size = { 420, 239 };
```

Comment:
This is way more than the two new widgets need. I suggest:
```suggestion
        static const Gfx::ui_size_t _window_size = { 420, 159 };
```